src/main/scala/com/ubirch/controllers/InfoController.scala

```diff
@@ -1,17 +1,18 @@
 package com.ubirch.controllers
 
-import com.typesafe.config.Config
 import com.ubirch.ConfPaths.GenericConfPaths
 import com.ubirch.controllers.concerns.{ ControllerBase, SwaggerElements }
-import com.ubirch.models.{ Good, NOK }
+import com.ubirch.models.{ NOK, Return }
+
+import com.typesafe.config.Config
 import io.prometheus.client.Counter
-import javax.inject._
 import monix.eval.Task
 import monix.execution.Scheduler
 import org.json4s.Formats
 import org.scalatra._
 import org.scalatra.swagger.{ Swagger, SwaggerSupportSyntax }
 
+import javax.inject._
```

Comment:
We also need to decide which inject we use. In some project we are using javax.inject but for other we are using google guice inject. They can work together with no problem but for consistency  it should be the same

Reply:
the one we want is the interface. Meaning, javax.